data.sql

```diff
@@ -35,3 +35,149 @@ UPDATE animals SET owner_id = 4 WHERE name LIKE 'Squirtle';
 UPDATE animals SET owner_id = 5 WHERE name LIKE 'Boarmon';
+
+INSERT INTO vets VALUES (DEFAULT, 'William Tatcher', 45, '04-23-2000');
+INSERT INTO vets VALUES (DEFAULT, 'Maisy Smith', 26, 'Jan-17-2019');
+INSERT INTO vets VALUES (DEFAULT, 'Stephanie Mendez', 64, 'May-04-1981');
+INSERT INTO vets VALUES (DEFAULT, 'Jack Harkness', 38, 'Jun-08-2008');
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'William Tatcher'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'), 
+  (SELECT id FROM species WHERE species.name = 'Digimon')
+);
+
+INSERT INTO specializations VALUES (
+  (SELECT id FROM vets WHERE vets.name = 'Jack Harkness'), 
+  (SELECT id FROM species WHERE species.name = 'Pokemon')
+);
+
+-- visits data ======================================================
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Agumon'),
+  (SELECT id FROM vets WHERE vets.name = 'William Tatcher'),
+  'May-24-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Agumon'),
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'),
+  'Jul-22-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Gabumon'),
+  (SELECT id FROM vets WHERE vets.name = 'Jack Harkness'),
+  'Feb-02-2021'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Pikachu'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'Jan-05-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Pikachu'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'Mar-08-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Pikachu'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'May-14-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Devimon'),
+  (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez'),
+  'May-04-2021'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Charmander'),
+  (SELECT id FROM vets WHERE vets.name = 'Jack Harkness'),
+  'Feb-24-2021'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Plantmon'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'Dec-21-2019'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Plantmon'),
+  (SELECT id FROM vets WHERE vets.name = 'William Tatcher'),
+  'Aug-10-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE animals.name = 'Plantmon'),
+  (SELECT id FROM vets WHERE vets.name = 'Maisy Smith'),
+  'Apr-07-2020'
```

Comment:
- [ ] @Vgonma kindly ensure you date is correct, its suppose to be Plantmon visited Maisy Smith on Apr 7th, 2021.